drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/inspectors/RuleInspector.java

```diff
@@ -64,8 +69,8 @@
     private final InspectorList<PatternInspector>            patternInspectorList    = new InspectorList<>();
     private final InspectorList<ConditionInspector>          brlConditionsInspectors = new InspectorList<>(true);
     private final InspectorList<ActionInspector>             brlActionInspectors     = new InspectorList<>(true);
-    private final InspectorList<ActionsInspectorMultiMap>    actionsInspectors       = new InspectorList<>( true );
-    private final InspectorList<ConditionsInspectorMultiMap> conditionsInspectors    = new InspectorList<>(true);
+    private       InspectorList<ActionsInspectorMultiMap>    actionsInspectors       = null;
+    private       InspectorList<ConditionsInspectorMultiMap> conditionsInspectors    = null;
```

Comment:
The immutable final field is definitely better. Why set this to null? It's initialized in the constructor anyway (unconditionally).